website/docs/guides/0.9.1-upgrade-guide.html.markdown

```diff
@@ -72,6 +72,55 @@ configuration and real physical resources that exist. As a result, no
 actions need to be performed.
 ```
 
+## Upgrading private endpoint and private endpoint interface link
+
+To upgrade from v0.8.0 to v0.9.1 with the existing resources `mongodbatlas_private_endpoint` and `mongodbatlas_private_endpoint_interface_link`
+perform the following steps
+
+1. Remove your existing `mongodbatlas_private_endpoint` and `mongodbatlas_private_endpoint_interface_link` resources from terraform state
+   **NOTE** before doing any terraform state change please do a backup
+
+```bash
+$ terraform state rm mongodbatlas_private_endpoint.test
+$ terraform state rm mongodbatlas_private_endpoint_interface_link.test
+```
+
+2. Change the version of provider to the newest version
```

Comment:
Let's say the version 0.9.1 , because in theory for v1 because is a major version old deprecated resources should be removed finally